code/_globalvars/lists/names.dm

```diff
@@ -12,9 +12,11 @@ GLOBAL_LIST_INIT(first_names_male, world.file2list("strings/names/first_male.txt
 GLOBAL_LIST_INIT(first_names_female, world.file2list("strings/names/first_female.txt"))
 GLOBAL_LIST_INIT(last_names, world.file2list("strings/names/last.txt"))
 GLOBAL_LIST_INIT(lizard_names_male, world.file2list("strings/names/lizard_male.txt"))
-GLOBAL_LIST_INIT(lizard_names_female, world.file2list("strings/names/lizard_female.txt"))
-GLOBAL_LIST_INIT(squid_names_male, world.file2list("strings/names/squid_male.txt"))
+GLOBAL_LIST_INIT(lizard_names_female, world.file2list("austation/strings/names/lizard_female.txt"))
```

Comment:
This needs to be changed back to "strings/names/lizard_female.txt"

Reply:
fixed, thanks